com.eclipsesource.modelserver.emf/src/test/java/com/eclipsesource/modelserver/emf/common/SessionControllerTest.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2019 EclipseSource and others.
+ *
+ *   This program and the accompanying materials are made available under the
+ *   terms of the Eclipse Public License v. 2.0 which is available at
+ *   http://www.eclipse.org/legal/epl-2.0.
+ *
+ *   This Source Code may also be made available under the following Secondary
+ *   Licenses when the conditions for such availability set forth in the Eclipse
+ *   Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ *   with the GNU Classpath Exception which is available at
+ *   https://www.gnu.org/software/classpath/license.html.
+ *
+ *   SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ *******************************************************************************/
+package com.eclipsesource.modelserver.emf.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import org.junit.Test;
+
+import io.javalin.websocket.WsContext;
+
+public class SessionControllerTest {
+
+	@Test
+	public void subscription() {
+		SessionController sessionController = new SessionController();
+
+		final WsContext client = mock(WsContext.class);
+		when(client.send(anyString())).thenReturn(CompletableFuture.completedFuture(null));
+		when(client.getSessionId()).thenReturn(UUID.randomUUID().toString());
+		when(client.pathParam("modeluri")).thenReturn("fancytesturi");
+		sessionController.subscribe(client, client.pathParam("modeluri"));
+
+		assertTrue(sessionController.isClientSubscribed(client));
+
+		sessionController.unsubscribe(client);
```

Comment:
Extract unsubscribe assert into own test. Some more test would be good (e.g. if `modelChanged` is called upon change/deletion), but let's add them via a separate PR.

Reply:
Extracted the test for unsubscribe for now and added an issue for extending unit tests (#26)